src/instructlab/training/main_ds.py

```diff
@@ -335,7 +337,7 @@ def train(
         )
 
     global_grad_norm = None
-    for epoch in range(args.num_epochs):
+    for epoch in range(args.current_epoch, args.num_epochs):
```

Comment:
Needs to be 
```suggestion
    for epoch in range(args.current_epoch, args.num_epochs+1):
```
 
Since `range(start, stop)` only goes from `start, ..., stop - 1`